tools/browsers-lists/browsers-list.mjs

```diff
@@ -1,26 +1,30 @@
 import { parseSpecString } from '../browser-matcher/spec-parser.mjs'
 import SpecMatcher from '../browser-matcher/spec-matcher.mjs'
+import browsersSupported from './browsers-supported.json' assert { type: "json" }
```

Comment:
What version of Node do the GH actions run in?  Can they support this syntax?

Reply:
Latest LTS version.